### ISSUE TITLE
Riot Client

### ISSUE THREAD

**Issue by DoisPontinhos** [sidit77/LoLAccountManager#2]
Hello, would it be possible to login with riot client closed? So the program auto opens riot client?

**Comment by sidit77** [comment]
Should be possible to do something like:

- start RiotClient.exe
- wait 5 seconds
- type username / password

However there's a lot that can go wrong with this:

- What if the client takes longer than 5s to load?
- What if the user recently closed the client and it can't be reopened yet?
- What if the client does not start with the login fields but some strange popup so something similar?
- What if the client is already open?

In short I think it's a better user experience to just double click both items, wait until the client has finished loading and then press an account button. Also keep in mind that this program already has to be manually updated by both by me and the user every time riot slightly moves the login fields around, adding even more stuff that can randomly break until a new version is released is imo not worth it.
